src/main/java/frc/robot/RobotMap.java

```diff
@@ -7,13 +7,17 @@
 
 package frc.robot;
 
+import edu.wpi.first.wpilibj.command.Subsystem;
+
 /**
  * The RobotMap is a mapping from the ports sensors and actuators are wired into
  * to a variable name. This provides flexibility changing wiring, makes checking
  * the wiring easier and significantly reduces the number of magic numbers
  * floating around.
  */
 public class RobotMap {
+  public static final int joystickNumber = 0;
+  public static final Subsystem m_actuator = null;
```

Comment:
A subsystem should not be on the robot map. I think you didn't mean for this line to be there.